RxCocoa/iOS/Proxies/RxDelegateProxyCrashFix.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  RxDelegateProxyCrashFix.swift
+//  RxCocoa
+//
+//  Created by SlashDevSlashGnoll (SlashDevSlashGnoll@users.noreply.github.com) on 9/9/22.
+//  Copyright © 2022 Krunoslav Zaher. All rights reserved.
+//
+
+import Foundation
+
+
+// This extension exists solely to get around a crash found on iOS 15.4+ where a `text`
+// method invocation is being sent to the RxCollectionViewDelegateProxy which doesn't implement it.
+// This is tracked at this bug: https://github.com/ReactiveX/RxSwift/issues/2428 This can be
+// removed if/when the actual source of the problem is found
+@objc extension RxCollectionViewDelegateProxy {
```

Comment:
did you bump into this only for Table / Collection or delegate proxies in general? 
If it's broader you could theoretically add this to DelegateProxy itself as an extension.
Thoughts?

Reply:
The reports we discussed only listed crashes for TableView proxy and CollectionView proxy so it seemed specific to those as far as I've seen.